internal/provider/config_utils.go

```diff
@@ -19,3 +19,24 @@ func boolFromConfig(config map[string]tftypes.Value, key string) bool {
 	config[key].As(&val)
 	return val
 }
+
+func stringSliceFromConfig(config map[string]tftypes.Value, key string) ([]string, error) {
```

Comment:
Could there be a better name for this? Slice is the operation it is doing internally but what is it actually returning? It's hard to know what this does without grokking all the code.

Reply:
It's returning a "string slice" from the config map.